data_steward/tools/retract_data_gcs.py

```diff
@@ -169,7 +193,6 @@ def get_response():
 def get_integer(num_str):
```

Comment:
I looked at `get_integer(num_st)` and I'm curious.  What does placing this as a function offer over an in line `int(num_str)` cast?

Reply:
It was performing other checks earlier, but we cut it down. I guess it's retained in case we want the functionality back.

Reply:
Actually, I'll remove this function since it doesn't serve any additional purpose for now